### ISSUE TITLE
IMOS : check_data_variables - history variable requires one of XYZT variable

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#33]
imos checker output:

```
history                        :3:     0/ 1 :  
            check_dimension            :3:     0/ 1 : dimension doesn't contain
                                                      TIME, LATITUDE, LONGITUDE,
                                                      DEPTH
```

ncdump

```
char history(h_num, h_string) ;
                history:long_name = "file history information" ;
```

file http://thredds.aodn.org.au/thredds/catalog/IMOS/SOOP/SOOP-ASF/VNAA_Aurora-Australis/flux_product/2013/catalog.html?dataset=IMOS/SOOP/SOOP-ASF/VNAA_Aurora-Australis/flux_product/2013/IMOS_SOOP-ASF_FMT_20130315T000200Z_VNAA_FV02.nc

The history variable is required to have a XYZT dimension. I don't think this should be the case. Maybe a black list of variables could be created (in a text file) ?

**Comment by mhidas** [comment]
I'd call this a bug. `history` is not a data variable, so this check shouldn't apply. This needs to be fixed in the helper function that identifies data variables.

**Comment by fxmzb123** [comment]
@mhidas based on the definition of data variable, history is not coordinate and ancillary variable. Do you think this is a special case for data variable?

**Comment by mhidas** [comment]
@fxmzb123  Let's add to the definition of a data variable the condition that it needs to have numeric type. This would exclude the `history` variable.

**Comment by fxmzb123** [comment]
@mhidas fixed and see pull request #52
